go/vt/vtgate/planbuilder/testdata/union_cases.txt

```diff
@@ -832,50 +760,6 @@ Gen4 plan same as above
     ]
   }
 }
-{
-  "QueryType": "SELECT",
-  "Original": "select * from music where id = 1 union select * from user where id = 1",
-  "Instructions": {
-    "OperatorType": "Distinct",
-    "Inputs": [
-      {
-        "OperatorType": "Concatenate",
-        "Inputs": [
-          {
-            "OperatorType": "Route",
-            "Variant": "SelectEqualUnique",
-            "Keyspace": {
-              "Name": "user",
-              "Sharded": true
-            },
-            "FieldQuery": "select * from music where 1 != 1",
-            "Query": "select distinct * from music where id = 1",
-            "Table": "music",
-            "Values": [
-              1
-            ],
-            "Vindex": "music_user_map"
-          },
-          {
-            "OperatorType": "Route",
-            "Variant": "SelectEqualUnique",
-            "Keyspace": {
-              "Name": "user",
-              "Sharded": true
-            },
-            "FieldQuery": "select * from `user` where 1 != 1",
-            "Query": "select distinct * from `user` where id = 1",
-            "Table": "`user`",
-            "Values": [
-              1
-            ],
-            "Vindex": "user_index"
-          }
-        ]
-      }
-    ]
-  }
-}
```

Comment:
Is this query no longer supported?

Reply:
Florent is working on `*`. As of this PR, we need to know the columns to plan an UNION